src/SignalR/clients/ts/FunctionalTests/ts/HubConnectionTests.ts

```diff
@@ -634,16 +634,16 @@ describe("hubConnection", () => {
             });
 
             if (transportType !== HttpTransportType.LongPolling) {
-                it("terminates if no messages received within timeout interval", (done) => {
+                it("terminates if no messages received within timeout interval", async (done) => {
                     const hubConnection = getConnectionBuilder(transportType).build();
                     hubConnection.serverTimeoutInMilliseconds = 100;
 
-                    hubConnection.start().then(() => {
-                        hubConnection.onclose((error) => {
-                            expect(error).toEqual(new Error("Server timeout elapsed without receiving a message from the server."));
-                            done();
-                        });
+                    hubConnection.onclose((error) => {
+                        expect(error).toEqual(new Error("Server timeout elapsed without receiving a message from the server."));
+                        done();
                     });
+
+                    await hubConnection.start();
```

Comment:
Is this await necessary given the test explicitly calls done()? Either way, LGTM. Just curious.

Reply:
I don't think it is, but it looks nicer to me, don't like not awaiting things that can be awaited.